CAT/5.3/Dockerfile

```diff
@@ -0,0 +1,87 @@
+# Set global variables
+ARG CAT_VER="5.3"
+ARG DIAMOND_VER="2.1.9"
+
+# Build Stage
+FROM ubuntu:focal AS builder
+ARG CAT_VER
+ARG DIAMOND_VER
+
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="CAT"
+LABEL software.version=${CAT_VER}
+LABEL description="CAT: a tool for taxonomic classification of contigs and metagenome-assembled genomes (MAGs)."
+LABEL website="https://github.com/dutilh/CAT"
+LABEL license.url="https://github.com/dutilh/CAT/blob/master/LICENSE.md"
+LABEL maintainer="Taylor K. Paisie"
+LABEL maintainer.email='ltj8@cdc.gov'
```

Comment:
Can you move your labels to the app stage? They get lost when you switch away from the builder stage.